### ISSUE TITLE
Couldn't export the shape created by joystick'n sliders

### ISSUE THREAD

**Issue by tianrandesign** [airbnb/lottie-web#1142]
Hey, 
I tried to export a project made with joystick'n sliders from bodymovin, I selected "Hidden" and "Guides" in the export settings. But still couldn't get the blue dashed square and the shape of joystick in the SVG file. There was only the path I drew for the head. Don't know which part I did wrong. I'm using Version 5.1.20    and AE CC 2018
 
What I wanted to do is as the example provided here:
https://codepen.io/airnan/pen/EmdXGo/?editors=0010

Thank you so much for helping!
Here is the aep file.
[self portrait2.aep.zip](https://github.com/airbnb/lottie-web/files/2181149/self.portrait2.aep.zip)

I tried to change the two layers that JnS gave me into non-guided layer, then they were able to be exported. But still couldn't let the joystick follow the mouse pointer. :(

Here is the file after this try
[head.zip](https://github.com/airbnb/lottie-web/files/2181995/head.zip)

**Comment by tianrandesign** [comment]
I tried to change the two layers that JnS gave me into non-guided layer, then they were able to be exported. But still couldn't let the joystick follow the mouse pointer. :(

Here is the file after this try
[head.zip](https://github.com/airbnb/lottie-web/files/2181995/head.zip)

**Comment by bodymovin** [comment]
Hi, how are you trying to make the joystick follow the mouse?

**Comment by tianrandesign** [comment]
hey, 

Thank you for replying! 
I finally got the mouse following solution from this one:  
[https://github.com/airbnb/lottie-web/issues/463]

But I'm still confused about the exporting guide layer issue. I have selected "Hidden" and "Guides" in the export settings, but still couldn't export the blue dashed square and the shape of joystick into the SVG file. What I did was right clicking on the layers to uncheck the "guide layer" option. Is this the right way?
